lua/lspconfig/fsautocomplete.lua

```diff
@@ -0,0 +1,39 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.fsautocomplete = {
+  default_config = {
+    root_dir = util.root_pattern('*.sln', '*.fsproj', '.git');
+    filetypes = {'fsharp'};
+    init_options = {
+      AutomaticWorkspaceInit = true;
+    };
+  };
+  docs = {
+    description = [[
+https://github.com/fsharp/FsAutoComplete
+
+Language Server for F# provded by FsAutoComplete (FSAC).
+
+Download a release of FsAutoComplete from [here](https://github.com/fsharp/FsAutoComplete/releases).
+It can also be built from sources. [Instructions](https://github.com/fsharp/FsAutoComplete) are on the main page.
```

Comment:
Maybe just "instruction to compile from source are found on the main repository.